content/home/src/howto_guides/submit2nmdc.md

```diff
@@ -110,11 +128,99 @@ A ‘Jump to column’ feature is available for searching for specific metadata
+
+The Field Notes App requires ORCiD authentication to access. 
+
+Submissions via Field Notes are available in the Submission Portal as an automatic backup when connected to wifi. The app is available for offline use, but requires internet connection to **start** a submission. 
+
+
+TODO: Update all these to field notes!
+
+(Step 1 of 3) Welcome to the NMDC Field Notes mobile app! 
+
+[![](../_static/images/howto_guides/portal_guide/Create_submission.png)](../_static/images/howto_guides/portal_guide/Create_submission.png)
+
+Once signed in with an ORCiD, you will have the ability to access existing studys or create a study using the 'New’ button. Details about each section are outlined below. Users can also return to existing submissions saved under the ORCiD account to resume their work at any point. 
+
+### New Study
+
+[![](../_static/images/howto_guides/portal_guide/study_info.png)](../_static/images/howto_guides/portal_guide/study_info.png)
+
+The Study Information page requires a valid ‘Study Name’ and a valid email address. All other fields are optional and can be completed in the app or in the Submission Portal. Study names and all other information can be updated or added via the app or Submission Portal. See [header](#submission-portal-functionality) [header](#study) for more details about Study information.
+
+Also in the New Study you will select your "Environment Template". This will correspond to the GSC Environmental Extension. More information is available above in sections [header](#environmental-package). Multiple environments can be selects and submitted in one submission.
+
+Select "Save" once study information has been complete.
+
+### Select Fields
+
+TODO: Add Image
+
+The Field Notes App allows the submitter to select the metadata terms (TODO what word?!) that will be visible and available on the app and when working in the field. 
+
+Available terms can be selected and de-selected individually or in bulk based on the section they're assigned to. The available sections are: "Common", terms commonly measured at the time of sample collection; "Occasional", terms that may sometimes be measured at the time of sample collection; "Uncommon", terms that are rarely measured at the time of sample collection. This allows the submitter to tailor their metadata interface to be more focused for field collection.
+
+If you are completing metadata for multiple sample types in multiple templates you will need to select your fields for each individual template. 
+We recommend completing all "New Study" steps prior to and in preparation of field work.
+
+### Samples
+
+In Field Notes, sample metadata are entered individually in a form like structure. This is different than the grid format of the Submission Portal
+
+The Sample (TODO Biosample in Submission Portal vs Sample here?) Metadata interface consists of a form like structure where metadata is entered individually, unlike the grid format of the Submission Portal.
```

Comment:
I would maybe only keep this one ("The sample ... Portal") or the previous one ("In Filed Notes ... Portal"), but not both ?

Reply:
@simroux  I don't follow you herE?

Reply:
I think this was just to say that I don't think we need both sentences, so we should just pick one (but I don't have a strong feeling on which) ?